### ISSUE TITLE
Support Gentoo

### ISSUE THREAD

**Issue by rzr** [adoptware/pinball#31]
I noticed that gentoo package is behind latest release:

https://github.com/gentoo/gentoo/tree/master/games-arcade/emilia-pinball

May this be bumped to latest release (on SDL1)  and then I'll release a SDL2 version that will ship GLESv1 from @caramelli and tested by @lanodan

**Comment by lanodan** [comment]
CC: @Polynomial-C as the maintainer of the package (which I could co-maintain btw)

I pushed `games-arcade/emilia-pinball-9999::lanodanOverlay` (webview: <https://hacktivis.me/git/overlay/file/games-arcade/emilia-pinball/emilia-pinball-9999.ebuild.html>) which targets the git repo, it depends on https://github.com/gentoo/gentoo/pull/20690 for the gles1 part.

The latest version is `0.3.20201218` right?

**Comment by rzr** [comment]
yes I plan to release 0.4.0 afterward. I just wanted a couple of changes I am polishing.

good luck with packaging

**Comment by Polynomial-C** [comment]
> CC: @Polynomial-C as the maintainer of the package (which I could co-maintain btw)
> 
> I pushed `games-arcade/emilia-pinball-9999::lanodanOverlay` (webview: https://hacktivis.me/git/overlay/file/games-arcade/emilia-pinball/emilia-pinball-9999.ebuild.html) which targets the git repo, it depends on [gentoo/gentoo#20690](https://github.com/gentoo/gentoo/pull/20690) for the gles1 part.
> 
> The latest version is `0.3.20201218` right?

Well, I have put some releases from [sergiomb2/pinball](https://github.com/sergiomb2/pinball) into my personal Gentoo ebuild overlay but not into official Gentoo ebuild repositiry. Of course, once https://github.com/gentoo/gentoo/pull/20690 has been merged, I can add latest release from here.

**Comment by rzr** [comment]
Update: https://mastodon.social/@rzr/109893196602425045
